android/library/src/main/java/com/liferay/mobile/screens/auth/login/LoginListener.java

```diff
@@ -31,4 +32,11 @@
 	 * @param e exception
 	 */
 	void onLoginFailure(Exception e);
+
+
+	/**
+	 * Called when the browser is opened to authenticate.
+	 * {@link AuthenticationType} should be OAUTH2REDIRECT
+	 */
+	void onAuthenticationBrowserShown();
```

Comment:
I would like to have default methods in Android...

Reply:
I would use a default method, but without changing all the lambdas (for a later PR)